mock-abis-default.properties

```diff
@@ -7,15 +7,15 @@ spring.application.name=mock-abis-service
 management.endpoint.health.show-details=always
 management.endpoits.web.exposure.include=info,health,refresh
 server.port=8081
-abis.return.duplicate=false
+abis.return.duplicate=true
```

Comment:
what should be the default value, as in the master branch it is set to "false" but now changed to "true" @JanardhanBS-SyncByte